content/de/verein/event/fireshonks-tracking-2022/index.md

```diff
@@ -7,7 +7,7 @@
         "online": "https://streaming.media.ccc.de/jev22/fireshonks"
     },
     "featured_image": "tracking-in-apps-ist-das-legal.png",
-    "license": "cc-by-40",
+    "video_url": "https://media.datenanfragen.de/embed/20221229-fireshonks",
```

Comment:
Can we add a thumbnail? Atm. it's just a black screen until you press play:

![image](https://user-images.githubusercontent.com/45886910/212311239-8a3969df-51df-4b8a-a0fd-165b89300b5c.png)

The hacky solution would be to add a thumbnail as first frame? :D

Reply:
Please don't make me replace the video file. :D 

I agree it would make sense to add thumbnail support but that needs to be implemented in datenanfragen/media. Feel free to make a PR. :P